portmap/core/admin.py

```diff
@@ -61,3 +63,20 @@ class CustomUserAdmin(UserAdmin):
 @register(DataType, site=admin_site)
 class DataTypeAdmin(admin.ModelAdmin):
     list_display = ('id', 'name', 'helpText')
+
+@register(TrackArticleView, site=admin_site)
+class TrackArticleViewAdmin(admin.ModelAdmin):
+    list_display = ('article_path', "count", "visited_directly", "external_referrer",)
+    aggregated_counts = {}
+
+    def count(self, object):
```

Comment:
Cool isnt it??? This 'count' column would be even more appropriate added to the Article admin list display view, becuase then the sum number would show up once per Article.

Reply:
That would be a better implementation, for sure.